src/Checkbox.jsx

```diff
@@ -29,15 +29,12 @@ export default class Checkbox extends React.Component {
   };
   constructor(props) {
     super(props);
-    let checked = false;
-    if ('checked' in props) {
-      checked = props.checked;
-    } else {
-      checked = props.defaultChecked;
-    }
+
+    const checked = 'checked' in props ? props.checked : props.defaultChecked;
+
```

Comment:
Because the defaultChecked in defaultProps is false,so we don't assign `false` to it.So the onditional operator makes this more clear.